CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## v5.0.0 Breaking changes
+
+- `calculations`: Removed `getRadian` export
+- `data`: Removed `setField` export, use [`lodash.set(object, path, value)`](https://lodash.com/docs/4.17.15#set) instead
```

Comment:
some of these lodash functions are quite big, it might still be preferrable to keep old versions when lodash covers usecases we don't need (support for maps/arrays/sets in place of objects etc)

Reply:
- Especially `setField` is needed very rarely. Most of the time we work on shallow objects.
- `lodash.round` looks ok to me, no need to optimize  https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L5414
- `lodash.upperFirst` looks ok too https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L4941

We should keep an eye on what we do during render loops but besides that I don't think it's worth maintaining our own helpers doing stuff that is already covered in lodash and doesn't cause noticeable performance differences.